src/ts/tinymce.ts

```diff
@@ -331,5 +334,31 @@ export function getTinymceBaseConfig(page: string): object {
       },
     ],
     toolbar_sticky: true,
+    // specifying custom CSS for properly rendering MathJax in TinyMCE instance
+    content_style: 'mjx-assistive-mml { position: absolute !important; top: 0px; left: 0px; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0px 0px 0px !important; border: 0px !important; display: block !important; width: auto !important; overflow: hidden !important; user-select: none; }' +
```

Comment:
Where did you get this css from? It should be indicated in the comment above

Reply:
I basically just copied and pasted the CSS for these elements when you inspect these elements on viewing an experiment/resource. They appear to be inline CSS so I'm not sure exactly which style sheets they came from, but I'm assuming that they're CSS specifications from MathJax itself. Not exactly sure how to phrase this in the comment.

Reply:
well something along: "to update this css, look at a mathjax element in a page and copy the css from that. This css was copied with version 3.2.2"